samples/nrf9160/https_client/src/main.c

```diff
@@ -141,15 +139,18 @@ int tls_setup(int fd)
 void main(void)
 {
 	int err;
-	int fd;
+	int fd = -1;
```

Comment:
Is this change needed? There's no way `fd` is used uninitialized here.

Reply:
I think it was picked up by some general good programming guidelines, e.g. in case a later change to the code (or typo/mistake) ever left it uninitialised; but it is a fairly simple example, so changed back.